vispy/gloo/context.py

```diff
@@ -166,7 +166,12 @@ def flush_commands(self, event=None):
         """
         if self._do_CURRENT_command:
             self._do_CURRENT_command = False
-            self.shared.parser.parse([('CURRENT', 0)])
+            canvas = get_current_canvas()
+            if canvas and hasattr(canvas, '_backend'):
+                fbo = canvas._backend._vispy_get_fb_bind_location()
+            else:
+                fbo = 0
+            self.shared.parser.parse([('CURRENT', 0, fbo)])
```

Comment:
so I guess before the implied/effective call was `[('CURRENT', 0, 0)]`, even though it was written as `[('CURRENT', 0)]`?